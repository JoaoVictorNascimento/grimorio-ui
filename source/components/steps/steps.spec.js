import '../../../internals/test/helper';

import Steps from './steps-component';

const pageData = [
  { name: 'pass1', title: 'Criação de Campanha', isComplete: true },
  { name: 'pass2', title: 'Criação de grupo de anúncio', isComplete: false },
  { name: 'pass3', title: 'Adicionar anúncios', isComplete: false },
  { name: 'finish', title: 'Finalização', isComplete: false },
];

let value = 0;
describe('Steps component', () => {
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(<Steps data={pageData} current={pageData[value].name} />);
      expect(wrapper.length).toEqual(1);
    });
  });
});
