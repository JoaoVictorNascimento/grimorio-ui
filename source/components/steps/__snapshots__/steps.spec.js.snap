// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Steps component #render render correctly 1`] = `
"<Panel title={false} size=\\"medium\\" loading={false} accordion={false} open={true} highlight=\\"shadow\\">
  <div className=\\"steps\\">
    <div className=\\"stepsHolder stepsHolderActive\\">
      <span className=\\"stepsNumber\\">
        <Icon name=\\"check\\" className=\\"icon\\" size={16} align=\\"middle\\" />
      </span>
      <span>
        Criação de Campanha
      </span>
      <div className=\\"stepsHolderLine\\" />
    </div>
    <div className=\\"stepsHolder stepsCurrent\\">
      <span className=\\"stepsNumber currentNumber\\">
        2
      </span>
      <span>
        Criação de grupo de anúncio
      </span>
      <div className=\\"stepsHolderLine\\" />
    </div>
    <div className=\\"stepsHolder\\">
      <span className=\\"stepsNumber\\">
        3
      </span>
      <span>
        Adicionar anúncios
      </span>
      <div className=\\"stepsHolderLine\\" />
    </div>
    <div className=\\"stepsHolder\\">
      <span className=\\"stepsNumber\\">
        4
      </span>
      <span>
        Finalização
      </span>
    </div>
  </div>
</Panel>"
`;
